Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
+
+1. Check if Ceph Cluster monitoring is enabled
+
+    `$ kubectl get cephcluster rook-ceph -n rook-ceph -oyaml`
+
+    ```YAML
+    apiVersion: ceph.rook.io/v1
+    kind: CephCluster
+    metadata:
+    name: rook-ceph
+    namespace: rook-ceph
+    [...]
+    spec:
+    [...]
+    monitoring:
+        enabled: true
+        rulesNamespace: "rook-ceph"
+    [...]
+    ```
+
+    > **Note:** Follow [ceph-monitoring/prometheus-alerts](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md#prometheus-alerts) to enable Ceph Cluster monitoring.
+
+2. Check for label on namespace
+
+    `$ kubectl get namespace rook-ceph --show-labels | grep openshift.io/cluster-monitoring=true`
+
+    ```bash
+    rook-ceph   Active   15h   openshift.io/cluster-monitoring=true
+    ```
+
+    If not present, label the namespace with,
+
+    `$ kubectl label namespace rook-ceph "openshift.io/cluster-monitoring=true"`
+
+3. Check if service monitor is present
+
+    `$ kubectl get servicemonitor -n rook-ceph`
+
+    ```bash
+    NAME                          AGE
+    rook-ceph-mgr                 14h
+    ```
+
+4. Check if ceph-mgr pod is Running
```

Comment:
This seems like it belongs in a separate section such as "Troubleshoot Monitoring Issues". The section would give hints for why the monitoring might not be working. If the mgr is not running that will be one reason. There might be other reasons as well such as the service monitor missing.

Reply:
Agreed. Will add a separate section for this.

Reply:
> Agreed. Will add a separate section for this.

In this PR?

Reply:
Yes.